__mocks__/mock-form-template.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const FormTemplate = ({ schema: { title, label, description }, formFields, formOptions, FormSpy }) => {
```

Comment:
It's a mock for tests! :eye: However, we can define some rules to using of `label` or `title` because we are not so consistent.. but we can do it later. This PR just takes the old implementation and implements it a new way.